mobile/native/src/trade/order/mod.rs

```diff
@@ -123,6 +124,18 @@ impl Order {
             }
         }
     }
+
+    /// This returns the trader's margin once known (based on the execution price).
+    pub fn trader_margin(&self) -> Option<u64> {
+        let opening_price = self.execution_price()?;
+
+        // QUESTION: Is the leverage always the trader's leverage or the self_leverage?
```

Comment:
🔧 Can remove this if someone answers haha.

Reply:
I am confused by the question. `self` in this context is owned by the `trader`for me:
Since the order is by the trader (owned by the trader) that order's leverage is always the trader's leverage.

Reply:
I think I worded this poorly. Let me try again:

Does the `Order` always represent the view of the party holding it? Or is it always fixed to the trader's perspective?

From your answer I understand that it depends on who holds it.

In conclusion, let's remove this question if it's still there.